### ISSUE TITLE
Bragging rights

### ISSUE THREAD

**Issue by RicksBeehive** [GMDevinity/Flood#270]
The cash section isn't really used to see how much cash others have, it's used to gauge how good/accomplished a player is.

We should have some way of bragging, because I see a lot of people doing it in game, be it changing the Cash section to display the totla cash they've ever earned, or having a 

!showwealth Donkie
"Donkie has a Total Wealth of $69696969" <- In chat

**Comment by Donkie** [comment]
I don't get the point of this at all, could you try re-explain it?

**Comment by RicksBeehive** [comment]
Have Total Money Earned displayed somewhere ingame or on the leaderboards because people buy tokens and shit. It'd purely be to show off

I know its stored because its used in level calculation.

**Comment by Donkie** [comment]
I see, wouldn't it be enough to display it in the playerinfo panel?

**Comment by RicksBeehive** [comment]
that'd be cool

**Comment by Donkie** [comment]
I thought about it and why do we even display cash in the scoreboard. Maybe that could be earned money instead?

**Comment by RicksBeehive** [comment]
Yeah my main point was nobody gives a fuck how much cash somebody currently has, total earned it a lot cooler basically

**Comment by jeffreythe00** [comment]
I'd love to see my total cash earned :p. That could also be something that is displayed in the scoreboard. Perhaps feature the top 5 players too.

**Comment by jeffreythe00** [comment]
I meant to say loading screen, not scoreboard

**Comment by Donkie** [comment]
@GMDevinity/moderators @GMDevinity/founders anyone against changing scoreboard cash display from "current" to "total earned"?

**Comment by CounterCraftz** [comment]
i'd keep it just incase because players are randomly getting a shit ton of cash w/ very little playtime
https://devinity.drpitlazar.us/playerstats/76561198073903206

**Comment by DawsonDevinity** [comment]
Agreed.

**Comment by RicksBeehive** [comment]
@CounterCraftz That dude's cash earned is less than his current cash. Something's clearly wrong with the amount of cash he has or how we calculate cash earned. But if we changed it over it would go from $1M to $300k, the amount he's probably earned.

**Comment by CounterCraftz** [comment]
That cash was most likely given. Cash earned is calculated by subtotal after the round ends

**Comment by RicksBeehive** [comment]
looks like we need to fix that shit then
